src/main/java/com/ensao/gi5/lint/util/Utils.java

```diff
@@ -45,6 +46,10 @@ public static CompilationUnitWrapper getCompilationUnit(File file) {
         }
     }
 
+    public static int getLine(SimpleName simpleName){
```

Comment:
could be 
```java 
public static int getLine(Node node){
         return node.getBegin()
                 .map(p -> p.line)
                 .orElse(-1);
    }
``` 
By changing `SimpleName` by a more generic type `Node`, this method can be used for all the nodes, possible usages will be mentioned below.